02-peer-protocol.md

```diff
@@ -243,6 +242,28 @@ this input spends. Used to verify that the input is non-malleable.
 replaceability, and the same value should be used across implementations
 to avoid on-chain fingerprinting.
 
+#### Liquidity griefing
+
+When sending `tx_add_input`, senders have no guarantee that the remote node
+will complete the protocol in a timely manner. Malicious remote nodes could
+delay messages or stop responding, which can result in a partially created
+transaction that cannot be broadcast by the honest node. If the honest node
+is locking the corresponding utxo exclusively for this remote node, this can
+be exploited to lock up the honest node's liquidity.
+
+It is thus recommended that implementations don't lock utxos and actively
+reuse them in concurrent sessions, which guarantees that transactions created
```

Comment:
Nit: the wording is slightly confusing -- does the "don't" also apply to "actively reuse them" or not?

```suggestion
It is thus recommended that implementations keep UTXOs unlocked and actively
reuse them in concurrent sessions, which guarantees that transactions created
```

Reply:
Thanks, agreed! Done in https://github.com/niftynei/lightning-rfc/pull/12/commits/4c300e597464dbf110648edec387b5f8e6e8bfaa